CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Android Card Form Release Notes
+
+## unreleased
+
+* Fix autofill issue that causes multiple slashes to be displayed in the expiration date field.
```

Comment:
```suggestion
* Fix autofill issue that causes multiple slashes to be displayed in the expiration date field (resolves #88)
```

Just a small suggestion to help us and merchants track which version has the fix for this issue.